Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -507,9 +567,23 @@ namespace Babylon
         Dispose();
     }
 
-    Napi::Value NativeEngine::HomogeneousDepth(const Napi::CallbackInfo& info)
+    void NativeEngine::LoadYFlipProgram()
     {
-        return Napi::Value::From(info.Env(), bgfx::getCaps()->homogeneousDepth);
+        if (!bgfx::getCaps()->originBottomLeft)
+        {
+            // This is temporary until a Shader plugin is developed.
+            // Shader is loaded/created here, then passed to graphics Impl for use and destruction.
+            // Idealy, this shader would be created and managed by a 3rd entity (Shader plugin)
```

Comment:
I think this comment needs to be on the SetYFlipProgram function also.

Reply:
done.